test/service/test_drs.py

```diff
@@ -140,46 +141,63 @@ def test_drs(self):
                         # We expect a DRS object with an access URL
                         expected['access_methods'] = [
                             {
-                                'access_url': {
-                                    'url': ('https://org-hca-dss-checkout-prod.s3.amazonaws.com/'
-                                            'blobs/307.a72.eb6?foo=bar&et=cetera')
-                                },
+                                'access_url': {'url': 'https://org-hca-dss-checkout-prod.s3.amazonaws.com/'
+                                                      'blobs/307.a72.eb6?foo=bar&et=cetera'},
                                 'type': 'https'
+                            },
+                            {
+                                'access_url': {'url': 'gs://important-bucket/object/path'},
+                                'type': 'gs'
                             }
                         ]
                         self.assertEqual(drs_object, expected)
                         # self.assertEqual(sort_frozen(freeze(drs_object)), sort_frozen(freeze(expected)))
                     else:
-                        # DRS object should have an access ID
+                        # The access IDs are so similar because the mock tokens are the same
                         expected['access_methods'] = [
                             {
-                                'access_id': ('eyJleGVjdXRpb25faWQiOiAiOTViMWZjZDAtNThjMi00ZjJjLWJiNDgtMTNhZDg1NmMyNGZj'
-                                              'IiwgInN0YXJ0X3RpbWUiOiAxNTc1MzI0MzgxLjE5ODM4NjcsICJhdHRlbXB0cyI6IDB9')
+                                'access_id': 'KCd7ImV4ZWN1dGlvbl9pZCI6ICI5NWIxZmNkMC01OGMyLTRmMmMtYmI0OC0xM2FkODU2YzI0Z'
+                                             'mMiLCAic3RhcnRfdGltZSI6IDE1NzUzMjQzODEuMTk4Mzg2NywgImF0dGVtcHRzIjogMH0nLC'
+                                             'AnYXdzJyk',
```

Comment:
```suggestion
                                             'AnYXdzJyk',
                                             #  ^ … but they do differ.
```

Or extract the common part into a variable.

Reply:
I'm going with your first suggestion.